css-layout-classes/styles.css

```diff
@@ -0,0 +1,41 @@
+  * {
+    font-family: Arial, Helvetica, sans-serif;
+    margin: 5px 10px;
+  }
+
+  p {
+    margin: 10px 10px;
+  }
+
+  img {
+    width: 100%;
+  }
+
+  .row {
+    display: flex;
+    justify-content: center;
```

Comment:
you don't need the justify content here, since all rows will be filled and there is no space to center elements within.